pubsub/google/cloud/pubsub_v1/subscriber/policy/base.py

```diff
@@ -252,10 +247,10 @@ def drop(self, items):
         # Remove the ack ID from lease management, and decrement the
         # byte counter.
         for item in items:
-            if item.ack_id in self.managed_ack_ids:
-                self.managed_ack_ids.remove(item.ack_id)
+            try:
```

Comment:
A more performant version of this would be:

```python
if self.leased_messages.pop(item.ack_id, None) is not None:
    self._bytes -= item.byte_size
else:
    _LOGGER.debug('Item %s was not managed.', item.ack_id)
```

Reply:
Good call, done.